### DISCUSSION TITLE
Chaterino Window Widget for Xbox Game Bar

### DISCUSSION THREAD

**Discussion by stevenstallone** [Chatterino/chatterino2#5430] [Ideas]
Make a Windows Game Bar App/Widget to forward the Chatterino window into the Game Bar overlay which is independent of everything and the window can be pinned and has transparency.

I used to use a twitch chat window feature in Gamecaster. Gamecaster had an Game Bar app, it was essentially a twitch chat window much like Chatterino, you could type into chat as well. You could use this without actually using Gamecaster to stream. Xsplit enveloped Gamecaster so now you have to use the Xsplit version but with one bid difference, you have to stream in order for it to work.

If the Chatterino window could be forwarded to the Game Bar overlay through a GameBar App/Widget it would create an almost permanent, convenient, accessible, toggable, twitch chat window ever. You can bring it up on top of any full screen game, you can hide it with ease, the xsplit version can have transparency when you pin it on screen. Basically Chatterino could be open in the background all the time and you could access the chat window by bringing up the Game Bar overlay or pinning the window to the screen. The overlay operates independently to what ever is happening on screen, it doesn't matter if the game is in the middle of loading, you always have access and control to the twitch chat window.

**Comment by Felanbird** [comment]
Windows only features are _very_ unlikely to get any love, we already have a Windows only browser extension that we do not give any development to
